lab1.js

```diff
@@ -93,7 +119,20 @@ var nextAnimal;
 
 // your code goes here
 
-assert(nextAnimal, 'assign something to nextAnimal');
+var animalPick = function() {
+	var random = Math.random() * favoriteAnimals.length;
```

Comment:
Check out Math.floor and Math.ceiling. Those two methods will give you round numbers, and then you can just assign nextAnimal = favoriteAnimals[random].